### ISSUE TITLE
Parent cursor movement doesn't change directory correctly

### ISSUE THREAD

**Issue by flipsi** [kamiyaa/joshuto#570]
If I navigate to a sibling directory "directly", i.e. via `["parent_cursor_move_down"]` (`]`), the working directory seems not updated correctly. That means I can't open files and shells are spawned in the wrong, original directory (this is how I understood the issue).

This seems like a quite obvious bug.

If I navigate to a sibling directory "indirectly", i.e. via `["cd ..", "cursor_move_down", "open"]`, everything works as expected.

**Comment by flipsi** [comment]
For other users, here's a workaround: 

```
  # these don't work
  # { keys = ["["], commands = ["parent_cursor_move_up"] },
  # { keys = ["]"], commands = ["parent_cursor_move_down"] },

  # these do work as expected
  { keys = ["["], commands = ["cd ..", "cursor_move_up", "open"] },
  { keys = ["]"], commands = ["cd ..", "cursor_move_down", "open"] },
```

**Comment by kamiyaa** [comment]
Thanks @flipsi ! Will fix this shortly

**Comment by kamiyaa** [comment]
Should be fixed with 43b2522b89e0d60b70289c8160e347c8d5aa0bb8

@flipsi try it out and let me know
Once its confirmed, I will close this issue

**Comment by flipsi** [comment]
Awesome, thank you very much for fixing this in no time! :muscle: 

I tried and can confirm it works as I'd expect.
